service/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# LS docker file
+
+# Tomcat OS base image
+FROM artifacts.bcp.absa.co.za/docker.io/tomcat:9.0.62-jdk8-corretto
```

Comment:
I guess that this is our hardened image, and there is absolutely nothing sensitive in it? (Since this is OSS repo)

Reply:
Good question, I have to find out.

Reply:
Replaced with configurable image, the default being vanilla tomcat image.